accelerator/errors.h

```diff
@@ -51,9 +51,12 @@ extern "C" {
 typedef enum {
   SC_OK = 0, /**< No error */
 
-  SC_HTTP_OK = 200,          /**< HTTP response OK */
-  SC_HTTP_BAD_REQUEST = 400, /**< HTTP response, error when parsing request */
-  SC_HTTP_NOT_FOUND = 404,   /**< HTTP request not found */
+  SC_HTTP_METHOD_GET = 10,     /**< HTTP GET method */
```

Comment:
Why would you mix HTTP methods with status code?

Reply:
I thought put them here was appropriate. Perhaps they should be separate?

Reply:
So-called status code is obtained after certain operations are proceeded, but HTTP methods such as `GET` and `POST` are relatively different from status code. In fact, the HTTP methods are applied before something takes place.

Reply:
I see. I'll move them to another enum type.